2025-10-13 21:59:24,116 [processing](INFO): Initializing backseat process (pilot lrauv based on marl)
2025-10-13 21:59:27,331 [Logger](CRITICAL): Uncaught exception
Traceback (most recent call last):
  File "./backseat_app/main.py", line 111, in <module>
    main()
  File "./backseat_app/main.py", line 107, in main
    backseat_app.spin()
  File "./backseat_app/main.py", line 75, in spin
    self.marl_processor.compute_new_heading()
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/processing.py", line 134, in compute_new_heading
    new_heading = self.rl_tracking.newAction(self.target_address,self.target_range,self.lrauv_pose,self.lrauv_depth,self.target_timestamp)
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/utils.py", line 856, in newAction
    if self.initialized == False:
AttributeError: 'TargetTracking' object has no attribute 'initialized'
